src/mailtemplates/getting-started-new/getting-started-new.html.mustache

```diff
@@ -0,0 +1,165 @@
+      }
+      .signature {
+        padding: 0px 24px;
+      }
+      .footer {
+        font-size: 12px;
+        color: #3b3f44;
+        text-align: center;
+        padding: 24px;
+      }
+      .footer a {
+        color: #615BC2;
+        text-decoration: underline;
+      }
+    </style>
+  </head>
+    <body>
+        <span class="preheader">
+            Welcome to Unleash! You’ve been invited to your organization’s Unleash account – start experimenting with features today.
+        </span>
+        <div class="logo-bar">
+            <img src="https://cdn.getunleash.io/unleash_logo_600.png" alt="Unleash logo" />
+        </div>
+    <div class="container">
+      <div class="header">
+        <img src="https://cdn.getunleash.io/email-assets/email_header_welcome.png" alt="Welcome to Unleash" />
+      </div>
+      <div class="content">
+        <h1>Welcome to Unleash {{ name }}!</h1>
+        <p>
+          You have been invited to your organization’s Unleash account.
+        </p>
+        {{# passwordLink }}
+        <p>
+          Start using Unleash by clicking the sign-up link below. Follow the
+          steps provided to set your password and get started.
+        </p>
+        <p>
+          <a href="{{{ passwordLink }}}" target="_blank" rel="noopener noreferrer" style="display:inline-block; padding:6px 20px; margin-top:8px; background-color:#6C65E5; color:#ffffff !important; text-decoration:none; border-radius:6px; font-weight:bold; font-size:14px;">Set up account</a>
+        </p>
+        {{/ passwordLink }}
+        {{^ passwordLink }}
+        <p>
+          Start using Unleash by clicking the log in link below.
+        </p>
+        <p>
+          <a href="{{{ unleashUrl }}}" target="_blank" rel="noopener noreferrer" style="display:inline-block; padding:6px 20px; margin-top:8px; background-color:#6C65E5; color:#ffffff !important; text-decoration:none; border-radius:6px; font-weight:bold; font-size:14px;">Log in</a>
+        </p>
+        {{/ passwordLink }}
+      </div>
+      <div class="resources">
+        <h2>Useful resources</h2>
+        <p>Once you are up and running, you might want to take a look at the following resources:</p>
+        <ul>
+          <li><a href="https://docs.getunleash.io" target="_blank" rel="noopener noreferrer">Documentation</a></li>
+          <li><a href="https://github.com/unleash" target="_blank" rel="noopener noreferrer">Code examples</a></li>
+          <li><a href="https://docs.getunleash.io/reference/sdks" target="_blank" rel="noopener noreferrer">SDKs</a></li>
+        </ul>
+      </div>
+      <div class="signature">
+        <p>Best,<br/>Your Unleash team</p>
+      </div>
+      <div class="footer">
+        <p>
+          <a href="https://github.com/unleash" target="_blank">Follow us on GitHub</a><br />
+          &copy; {{ year }} Bricks Software – All rights reserved.<br />
+          Our mailing address is: <a href="mailto:team@getunleash.io">team@getunleash.io</a>
```

Comment:
Is this the email we want to share? It's received by everyone...

Reply:
Addressed in https://github.com/Unleash/unleash/pull/9980/commits/59bdaae1b88d26d59eba34b2a5289348613ca460